openapi.yaml

```diff
@@ -3578,7 +3580,6 @@ components:
                 remote_name: Finance Team
                 description: This group represents Active Directory group "Payments Production Admin". We use this AD group to facilitate staging deployments and qualifying new releases.
                 admin_owner_id: 7c86c85d-0651-43e2-a748-d69d658418e8
-                max_duration: 120
```

Comment:
nixed because `max_duration` should come from the request_configuration. top level field is deprecated